src/index.css

```diff
@@ -316,6 +321,52 @@ w3m-modal {
   white-space: nowrap;
 }
 
+/*  These classes are flawed in the version 4.11.1 of DaisyUI, so they are overwritten */
```

Comment:
'Flawed' in which sense? Just behaving differently than before or are they buggy?

Reply:
@ebma  They completely change previous designs, causing style bugs throughout the app

Reply:
Hmm okay, weird 🤔 but thanks for fixing it